analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -816,7 +816,8 @@ def parse_options(compilation_db_entry, compiler_info_file=None):
     details['compiler'] =\
         determine_compiler(gcc_command,
                            ImplicitCompilerInfo.is_executable_compiler)
-    if '++' in details['compiler'] or 'cpp' in details['compiler']:
+    compiler = os.path.basename(details['compiler'])
+    if '++' in compiler or 'cpp' in compiler:
```

Comment:
Hmm, do we see `cpp` in compiler names? This is hard, because `cpp` as in the command line utility stands for `C PreProcessor`.

Reply:
You're right, it must have been a wrong intuition to take it as a compiler, so I fixed it. By the way I don't even know if there is a build command where we have to fallback on compiler's name when determining the language. The source file name and `-x` flag should be enough.